specification/ai/HealthInsights/HealthInsights.RadiologyInsights/client.tsp

```diff
@@ -90,13 +90,13 @@ interface RadiologyInsightsClient {
 
 @@clientName(AzureHealthInsights.DocumentAuthor,
   "ClinicalDocumentAuthor",
-  "csharp"
+  "csharp, java"
```

Comment:
please also add javascript in scope

Reply:
@wanlwanl clientName is language specific, they don't need to add this for js.

Reply:
This PR is the config for Java SDK, on Java arch request.

If you need to config for TS, do a new one.